pkg/cache/azure_cache.go

```diff
@@ -88,8 +88,8 @@ type ResourceProvider struct {
 	Getter GetFunc
 }
 
-// NewTimedCache creates a new azcache.Resource.
-func NewTimedCache(ttl time.Duration, getter GetFunc, disabled bool) (Resource, error) {
+// NewCachedResourceRepo creates a new azcache.Resource.
+func NewCachedResourceRepo(ttl time.Duration, getter GetFunc, disabled bool) (Resource, error) {
```

Comment:
any reasons to rename this function? and what does 'Repo' mean here?

Reply:
repo is short for repository. I will raise another pr and replace timedcache with the one in client-go